tests/utilities/test_xla_device_utils.py

```diff
@@ -29,18 +28,23 @@ def test_tpu_device_absence():
 
 
 @RunIf(tpu=True)
-@pl_multi_process_test
 def test_tpu_device_presence():
     """Check tpu_device_exists returns True when TPU is available"""
-    assert xla_utils.XLADeviceUtils.tpu_device_exists() is True
+    assert xla_utils.XLADeviceUtils.tpu_device_exists()
 
 
-@patch('pytorch_lightning.utilities.xla_device.TPU_CHECK_TIMEOUT', 10)
+def sleep_fn(sleep_time: float) -> bool:
+    time.sleep(sleep_time)
+    return True
+
+
+@patch('pytorch_lightning.utilities.xla_device.TPU_CHECK_TIMEOUT', 3)
 def test_result_returns_within_timeout_seconds():
-    """Check that pl_multi_process returns within 10 seconds"""
+    """Check that pl_multi_process returns within 3 seconds"""
+
     start = time.time()
-    result = xla_utils.pl_multi_process(time.sleep)(xla_utils.TPU_CHECK_TIMEOUT * 1.25)
+    result = xla_utils.pl_multi_process(sleep_fn)(xla_utils.TPU_CHECK_TIMEOUT * 0.5)
```

Comment:
nit: in case one fails, easier to debug
```suggestion
    fn = xla_utils.pl_multi_process(sleep_fn)
    result = fn(xla_utils.TPU_CHECK_TIMEOUT * 0.5)
```